sympy/solvers/solvers.py

```diff
@@ -2453,24 +2481,33 @@ def solve_linear_system(system, *symbols, **flags):
 
 
 def solve_undetermined_coeffs(equ, coeffs, sym, **flags):
-    """Solve equation of a type p(x; a_1, ..., a_k) == q(x) where both
-       p, q are univariate polynomials and f depends on k parameters.
-       The result of this functions is a dictionary with symbolic
-       values of those parameters with respect to coefficients in q.
+    """
+    Solve equation of a type ``p(x; a_1, ..., a_k) == q(x)`` where both
```

Comment:
This should be math, not code (it isn't valid Python). `$p(x; a_1, \ldots, a_k) = q(x)$`. Same with the other variables in this sentence that refer to this.